### ISSUE TITLE
`Link` functionality broken

### ISSUE THREAD

**Issue by bradfrost** [pattern-lab/edition-php-development#17]
It looks like the `link` functionality is broken, both for links defined in Mustache code and those defined in JSON.

First the Mustache links:

```````
<a href="{{ link.pages-homepage }}">Test</a>
``````

Clicking the link returns a 404 "The requested URL `/04-pages-00-homepage/04-pages-00-homepage.html` was not found on this server."

It looks like the path is missing the necessary `/patterns/`, which is included in the path when you open a pattern in a new window.

And for the JSON link, I'm including a link using the following:

```````
"hero" : {
    "url" : "link.pages-blog"
}
``````

Clicking the link on the demo homepage returns a 404 "The requested URL `/patterns/04-pages-00-homepage/link.pages-blog` was not found on the server.

This looks like something different than the Mustache link, and I'm not sure what's exactly going on with it.

**Comment by dmolsen** [comment]
This should be fixed. Try running `composer update pattern-lab/core` to see if it pulls the latest changes. Had an issue with scope after I switched to getters and setters.

**Comment by bradfrost** [comment]
@dmolsen hmmmm no dice. Running `composer update pattern-lab/core` was successful, but the path is still omitting the `/patterns/` at the beginning. However, it now appears to be happening for both Mustache and JSON links.

**Comment by dmolsen** [comment]
I swear this issue had something else in it. I must be seeing things. So I "fixed" the wrong thing apparently. I'll try again. Sorry.

**Comment by bradfrost** [comment]
Trying to test this out again. I updated a test project, which updated successfully: 
![link-test bash 80x24](https://cloud.githubusercontent.com/assets/383701/11511700/92bb4cb2-9838-11e5-8b0b-13ca2a425b52.png)

I also did a clean install of a project, and yeah in both instances the `/patterns/` part of the path is missing from the links. No need to apologize; let me know what else I can provide to help fix it!
